CWE/events/disasters.txt

```diff
@@ -1148,3 +1148,26 @@ province_event = {
 		}
 	}
 }
+
+country_event = { 
+	id = 294
+	title = "Vajont Disaster" 
+	desc = "EVT_294_DESC" 
+	picture = "ashgabat_earthquake"
+	fire_only_once = yes
+	
+	trigger = {
+		year = 1963 NOT = { year = 1992 }
+		owns = 733 # Udine
+	}
+
+	mean_time_to_happen = {
+		months = 10
+	}
+	
+	option = {
+		name = "EVT_235_A"
```

Comment:
Please insert a comment for all option text so it’s easier for future modders to know what each event option is supposed to do.